src/status.js

```diff
@@ -1,23 +1,23 @@
 export default class Status {
   constructor() {
     if (localStorage.getItem('list') === null) {
-      this.list = {
-        0: {
+      this.list = [
+        {
           description: 'some description',
           completed: false,
           index: 0,
         },
-        1: {
+        {
           description: 'today\'s task',
           completed: false,
           index: 1,
         },
-        2: {
+        {
           description: 'yesterday\'s tast description',
           completed: false,
           index: 2,
         },
-      };
+      ];
```

Comment:
- [ ] For this milestone I should not see any hardcoded items of the array, which means, it should start with an empty To-Do list, kindly remove these items 😄